adding contact processing guide (#566)
publiccodenet/about: activities/events/process-contacts.md

# Process contacts after an event

## Add contacts to Odoo

If you agreed to keep in touch with someone at the event, add them as a contact in Odoo to enable that connection. Do not enter more details than what was shared with you for that purpose.

## Follow up

The way to follow up obviously depend on what interaction you had with the person on the event. If you had a good conversation with someone or heard them giving a good talk, consider sending them a brief thank you. Everyone that gives us inspiration in some way further our mission and deserves to be recognized. Other follow ups could include more information about the Foundation or the Standard or whatever you talked about. When you do follow up, include an [invitation to the mailing list](https://forms.gle/gn7wR2Eaxbv5g1BF9) to give them the option to subscribe to more news about us. Do not add them manually unless that was agreed upon explicitly. As we meet people for many different places we often play a role of a connector and introducing people to each other is a common way for us to add value. However, make sure to always get consent to do so before introducing people to each other.

## Share knowledge internally

> If some of the persons you met also are interesting to other staff members, brief them about your meeting and introduce each other in a medium that makes sense for the contact.

Reviewer: ```suggestion
If a person you met would be interesting to other staff members, brief them about your meeting and introduce each other in a medium that makes sense for the contact.
```
